strawberryfields/decompositions.py

```diff
@@ -22,21 +51,20 @@
 
 
 def takagi(N, tol=1e-13, rounding=13):
-    r"""Computes the Autonne-Takagi decomposition of a complex symmetric (not Hermitian!) matrix.
+    r"""Autonne-Takagi decomposition of a complex symmetric (not Hermitian!) matrix.
```

Comment:
```suggestion
    r"""Autonne-Takagi decomposition of a complex symmetric (not necessarily Hermitian) matrix.
```
@nquesada can you confirm this suggestion is ok?